fr.natron.Natron.json

```diff
@@ -0,0 +1,1079 @@
+                    "dest": "./"
+                },
+                {
+                    "type": "git",
+                    "url": "https://github.com/dtschump/gmic-community.git",
+                    "commit": "249ac90ac824eb99d233d8738f95658e92f270c9",
+                    "dest": "./gmic-community"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "cp ./gmic-community/libcgmic/gmic_libc.h ./",
+                        "cp ./gmic-community/libcgmic/gmic_stdlib_gmic.h ./",
+                        "cp ./gmic-community/libcgmic/gmic_libc.cpp ./"
+                    ]
+                },
+                 {
+                    "type": "patch",
+                    "path": "patches/openfx-gmic_plugin_path.patch"
+                }
+            ],
+            "build-commands": [
+                "make -j${MKJOBS} OPENMP=1 CONFIG=release PREFIX=/app",
+                "make install CONFIG=release DESTDIR=/app"
+            ],
+            "modules": [
+                {
+                    "name": "fftw3",
+                    "config-opts": [
+                        "--enable-threads",
+                        "--enable-sse2",
+                        "--enable-avx",
+                        "--enable-avx-128-fma",
+                        "--with-pic"
+                    ],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "http://www.fftw.org/fftw-3.3.7.tar.gz",
+                            "sha256": "3b609b7feba5230e8f6dd8d245ddbefac324c5a6ae4186947670d9ac2cd25573"
+                        }
+                    ],
+                    "cleanup": [
+                        "/include"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "natron",
+            "buildsystem": "qmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/NatronGitHub/Natron.git",
+                    "tag": "Natron-2.3.11",
+                    "commit": "fe372c98f2151b6451a75cd44343cd0505ea964d"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/natron_plugins-path.diff"
+                },
+                {
+                    "type": "file",
+                    "path": "config.pri"
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/NatronGitHub/OpenColorIO-Configs/archive/Natron-v2.3.tar.gz",
+                    "sha256": "0192bd27e467332c4e61c0103804198f941c88f43df7d9ba8ebb41b79adda640",
+                    "dest": "./OpenColorIO-Configs"
+                }
+            ],
+            "build-commands": [
+                "qmake -r CONFIG=stable"
```

Comment:
You shouldn't have both `build-commands` and `qmake` buildsystem. Use `config-opts` array to pass parameters to qmake.